libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3720,6 +3727,73 @@ string YulUtilFunctions::arrayConversionFunction(ArrayType const& _from, ArrayTy
 	});
 }
 
+string YulUtilFunctions::inlineArrayConversionFunction(InlineArrayType const& _from, ArrayType const& _to)
+{
+	if (_to.dataStoredIn(DataLocation::CallData))
+		solAssert(false);
+
+	if (!_to.isDynamicallySized())
+		solAssert(_to.length() == _from.components().size());
+
+	// TODO
+	// Other cases are done explicitly in LValue::storeValue, and only possible by assignment.
+	/*if (_to.location() == DataLocation::Storage)
+		solAssert(
+			(_to.isPointer() || (_from.isByteArrayOrString() && _to.isByteArrayOrString())) &&
+			_from.location() == DataLocation::Storage,
+			"Invalid conversion to storage type."
+		);
+	*/
+	string functionName =
+		"convert_inline_array_" +
+		_from.identifier() +
+		"_to_" +
+		_to.identifier();
+
+		vector<map<string, string>> memberSetValues;
+				unsigned stackItemIndex = 0;
+				for (auto&& [index, type]: _from.components() | ranges::views::enumerate)
+				{
+					memberSetValues.emplace_back();
+
+					memberSetValues.back()["setMember"] = Whiskers(R"(
+						{
+							let <memberValues> := <conversionFunction>(<value>)
+							<writeToMemory>(add(mpos, <offset>), <memberValues>)
+						}
+					)")
+					("memberValues", suffixedVariableNameList("memberValue_", stackItemIndex, stackItemIndex + _to.baseType()->stackItems().size()))
```

Comment:
```suggestion
					("memberValues", suffixedVariableNameList("memberValue_", 0, _to.baseType()->stackItems().size()))
```
Maybe I'm just a bit overzealous on having a look at each of your temporary pushes ;-) - but given that you now have a scope, you should be able to start at a zero suffix here each time.

Reply:
That is fine, all comments are highly appreciated :) Code updated accordingly.